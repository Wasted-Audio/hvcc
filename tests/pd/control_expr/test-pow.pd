#N canvas 827 239 734 565 12;
#X obj 40 22 loadbang;
#X obj 40 49 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 40 230 print;
#X msg 40 137 0.5;
#X msg 78 137 -2.5;
#X msg 124 137 3.7;
#X obj 40 194 expr pow($f1 \, $f2);
#X msg 165 137 5;
#X obj 40 80 t b b b b;
#X connect 0 0 1 0;
#X connect 1 0 8 0;
#X connect 3 0 6 0;
#X connect 4 0 6 1;
#X connect 5 0 6 0;
#X connect 6 0 2 0;
#X connect 7 0 6 1;
#X connect 8 0 3 0;
#X connect 8 1 4 0;
#X connect 8 2 5 0;
#X connect 8 3 7 0;
