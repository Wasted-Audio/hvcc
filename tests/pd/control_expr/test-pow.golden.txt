[@ 0.000] print: 693.44
[@ 0.000] print: 5.65685
